client/me/style.scss

```diff
@@ -30,6 +30,15 @@ body.is-section-me {
 		padding: 24px;
 	}
 
+	.masterbar__item-subitems {
+		background: var(--color-masterbar-item-active-background);
+
+		a:hover,
+		.masterbar__item-subitems-item:hover {
+			background: var(--color-masterbar-item-active-background);
+		}
+	}
+
```

Comment:
Instead of adding more styles and rules, we should just change the existing variable that controls this color `--color-global-masterbar-submenu-background`

Its defined here in the sidebar scss file: https://github.com/Automattic/wp-calypso/blob/453b8b2e3cdf43e2a763725a835b0e6ea7aeced0/client/my-sites/sidebar/style.scss#L68

Reply:
This would change the background for all the submenu dropdowns in the global area.  I doubt we would want them to be different colors from each other. However, I think the current white fits better personally. [I commented](https://github.com/Automattic/dotcom-forge/issues/8576#issuecomment-2312918720) on the issue.